src/Swap.java

```diff
@@ -0,0 +1,14 @@
+
+public class Swap {
+	static void swapNum(int num1, int num2) {
+		num2 = num1 + num2;
+		num1 = num2 - num1;
+		num2 = num2 - num1;
+		System.out.println(" Numbers After Swapping : " + (num1) + "," + num2);
```

Comment:
No need to put num1 in bracket here I think

Reply:
Thanks for suggestion, Changes Incorporated.